source/features/releases-tab.tsx

```diff
@@ -18,9 +18,9 @@ const repoUrl = getRepoURL();
 const repoKey = `releases-count:${repoUrl}`;
 
 // Get as soon as possible, to have it ready before the first paint
-const cached = cache.get(repoKey);
+const cached = cache.get<number>(repoKey);
```

Comment:
So the cache is generic now and it's fine. I misread the code before when I said it was always returning a string from the cache.

It actually does a `JSON.stringify` before storing the value in the cookie and a `JSON.parse` when it pulls it out of the cookie, so the stored data has the correct type.